vello_encoding/src/encoding.rs

```diff
@@ -402,7 +402,7 @@ impl Encoding {
 
     /// Encodes an image brush.
     pub fn encode_image(&mut self, image: &Image, alpha: f32) {
-        let _alpha = alpha * image.alpha;
+        let alpha = (alpha * image.alpha * 255.0).round() as u8;
```

Comment:
This is the canonical implementation (see e.g. https://github.com/linebender/svgtypes/pull/34).